cmd/lightclient/lightclient/subscriber.go

```diff
@@ -76,6 +76,12 @@ func (c *ChainTipSubscriber) handleGossipData(data *lightrpc.GossipData) error {
 		// Swap and replace
 		c.prevBlock = c.currBlock
 		c.currBlock = block.Block
+	case lightrpc.GossipType_AggregateAndProofGossipType:
```

Comment:
Note that you are adding this in cmd/lightclient/lightclient, we will handle this in a fullclient package later on, I probably should make separation of directories to make this clear. you can remove this, it is unnecessary for now.

Reply:
sure, removed!